Specification/4._CAN_bus_transport_layer.md

```diff
@@ -268,8 +264,8 @@ all available redundant interfaces simultaneously.
 
 An exception to the above rule is applicable if the payload of the transfer depends on some properties of the
 interface through which the transfer is emitted.
-An example of such special case is the time synchronization algorithm leveraged by UAVCAN
-(it is documented in the following sections of the specification).
+An example of such a special case is the time-synchronization algorithm leveraged by UAVCAN
```

Comment:
I couldn't find map to this algorithm below - ie not sure reading this where I might look for this example.

Reply:
Ah, a missing forward reference. The algorithm is documented in chapter 6.

Reply:
@pavel-kirienko Can I leave this reference for you to add please?